<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectWidget</class>
 <widget class="QMainWindow" name="ProjectWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Проект</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="centralwidgetHorizontalLayout">
    <item>
     <layout class="QHBoxLayout" name="mainHorizontalLayout" stretch="1,0,5">
      <item>
       <layout class="QVBoxLayout" name="leftVerticalLayout" stretch="0,0">
        <item>
         <widget class="QFrame" name="menuFrame">
          <property name="frameShape">
           <enum>QFrame::Panel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <layout class="QVBoxLayout" name="menuFrameVerticalLayout">
           <item>
            <widget class="QPushButton" name="backlogPushButton">
             <property name="font">
              <font>
               <family>GOST type B</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Backlog</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="renamePushButton">
             <property name="font">
              <font>
               <family>GOST type B</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Переименовать</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="addDeveloperPushButton">
             <property name="font">
              <font>
               <family>GOST type B</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Новый участник</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="editSprintPushButton">
             <property name="font">
              <font>
               <family>GOST type B</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Изменить спринт</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="developersGroupBox">
          <property name="font">
           <font>
            <family>GOST type B</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="title">
           <string>Участники</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <layout class="QVBoxLayout" name="developersGroupBoxVerticalLayout">
           <property name="leftMargin">
            <number>5</number>
           </property>
           <property name="topMargin">
            <number>5</number>
           </property>
           <property name="rightMargin">
            <number>5</number>
           </property>
           <property name="bottomMargin">
            <number>5</number>
           </property>
           <item>
            <widget class="QScrollArea" name="developersScrollArea">
             <property name="frameShape">
              <enum>QFrame::Panel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Sunken</enum>
             </property>
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="developersScrollAreaWidgetContents">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>177</width>
                <height>472</height>
               </rect>
              </property>
              <layout class="QVBoxLayout" name="developersScrollAreaWidgetContentsVerticalLayout"/>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="rightVerticalLayout" stretch="1,4">
        <item>
         <widget class="QLabel" name="projectLabel">
          <property name="font">
           <font>
            <family>GOST type B</family>
            <pointsize>25</pointsize>
           </font>
          </property>
          <property name="text">
           <string>New project</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="pinboardFrame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="pinboardFrameVerticalLayout" stretch="1,10">
           <item>
            <layout class="QHBoxLayout" name="sprintsHorizontalLayout" stretch="3,1,1">
             <item>
              <widget class="QComboBox" name="sprintsComboBox">
               <property name="font">
                <font>
                 <family>GOST type B</family>
                 <pointsize>12</pointsize>
                </font>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="startDateGroupBox">
               <property name="font">
                <font>
                 <family>GOST type B</family>
                 <pointsize>8</pointsize>
                </font>
               </property>
               <property name="title">
                <string>Начало</string>
               </property>
               <layout class="QHBoxLayout" name="startDateGroupBoxHorizontalLayout">
                <property name="leftMargin">
                 <number>3</number>
                </property>
                <property name="topMargin">
                 <number>3</number>
                </property>
                <property name="rightMargin">
                 <number>3</number>
                </property>
                <property name="bottomMargin">
                 <number>3</number>
                </property>
                <item>
                 <widget class="QLabel" name="startDateLabel">
                  <property name="font">
                   <font>
                    <pointsize>12</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>Start</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="endDateGroupBox">
               <property name="font">
                <font>
                 <family>GOST type B</family>
                 <pointsize>8</pointsize>
                </font>
               </property>
               <property name="title">
                <string>Конец</string>
               </property>
               <layout class="QHBoxLayout" name="endDateGroupBoxHorizontalLayout">
                <property name="leftMargin">
                 <number>3</number>
                </property>
                <property name="topMargin">
                 <number>3</number>
                </property>
                <property name="rightMargin">
                 <number>3</number>
                </property>
                <property name="bottomMargin">
                 <number>3</number>
                </property>
                <item>
                 <widget class="QLabel" name="endDateLabel">
                  <property name="font">
                   <font>
                    <pointsize>12</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>End</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="pinboardsHorizontalLayout" stretch="1,1,1,1,1">
             <item>
              <widget class="QGroupBox" name="todoGroupBox">
               <property name="font">
                <font>
                 <family>GOST type B</family>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="title">
                <string>To do</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
               <property name="flat">
                <bool>false</bool>
               </property>
               <property name="checkable">
                <bool>false</bool>
               </property>
               <layout class="QVBoxLayout" name="todoGroupBoxVerticalLayout">
                <property name="leftMargin">
                 <number>5</number>
                </property>
                <property name="topMargin">
                 <number>5</number>
                </property>
                <property name="rightMargin">
                 <number>5</number>
                </property>
                <property name="bottomMargin">
                 <number>5</number>
                </property>
                <item>
                 <widget class="QScrollArea" name="toDoScrollArea">
                  <property name="frameShape">
                   <enum>QFrame::Panel</enum>
                  </property>
                  <property name="widgetResizable">
                   <bool>true</bool>
                  </property>
                  <widget class="QWidget" name="toDoScrollAreaWidgetContents">
                   <property name="geometry">
                    <rect>
                     <x>0</x>
                     <y>0</y>
                     <width>168</width>
                     <height>419</height>
                    </rect>
                   </property>
                   <layout class="QVBoxLayout" name="toDoScrollAreaWidgetContentsVerticalLayout">
                    <item>
                     <widget class="QLabel" name="storyLabel">
                      <property name="minimumSize">
                       <size>
                        <width>0</width>
                        <height>40</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>16777215</width>
                        <height>16777215</height>
                       </size>
                      </property>
                      <property name="font">
                       <font>
                        <pointsize>12</pointsize>
                       </font>
                      </property>
                      <property name="frameShape">
                       <enum>QFrame::WinPanel</enum>
                      </property>
                      <property name="frameShadow">
                       <enum>QFrame::Raised</enum>
                      </property>
                      <property name="text">
                       <string>Story</string>
                      </property>
                      <property name="wordWrap">
                       <bool>true</bool>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <spacer name="toDoVerticalSpacer">
                      <property name="orientation">
                       <enum>Qt::Vertical</enum>
                      </property>
                      <property name="sizeHint" stdset="0">
                       <size>
                        <width>20</width>
                        <height>40</height>
                       </size>
                      </property>
                     </spacer>
                    </item>
                   </layout>
                  </widget>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="rejectedGroupBox">
               <property name="font">
                <font>
                 <family>GOST type B</family>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="title">
                <string>Rejected</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
               <layout class="QVBoxLayout" name="rejectedGroupBoxVerticalLayout">
                <property name="leftMargin">
                 <number>5</number>
                </property>
                <property name="topMargin">
                 <number>5</number>
                </property>
                <property name="rightMargin">
                 <number>5</number>
                </property>
                <property name="bottomMargin">
                 <number>5</number>
                </property>
                <item>
                 <widget class="QScrollArea" name="rejectedScrollArea">
                  <property name="frameShape">
                   <enum>QFrame::Panel</enum>
                  </property>
                  <property name="widgetResizable">
                   <bool>true</bool>
                  </property>
                  <widget class="QWidget" name="rejectedScrollAreaWidgetContents">
                   <property name="geometry">
                    <rect>
                     <x>0</x>
                     <y>0</y>
                     <width>168</width>
                     <height>419</height>
                    </rect>
                   </property>
                   <layout class="QVBoxLayout" name="rejectedScrollAreaWidgetContentsVerticalLayout"/>
                  </widget>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="inProgressGroupBox">
               <property name="font">
                <font>
                 <family>GOST type B</family>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="title">
                <string>In progress</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
               <layout class="QVBoxLayout" name="inProgressGroupBoxVerticalLayout">
                <property name="leftMargin">
                 <number>5</number>
                </property>
                <property name="topMargin">
                 <number>5</number>
                </property>
                <property name="rightMargin">
                 <number>5</number>
                </property>
                <property name="bottomMargin">
                 <number>5</number>
                </property>
                <item>
                 <widget class="QScrollArea" name="inProgressScrollArea">
                  <property name="frameShape">
                   <enum>QFrame::Panel</enum>
                  </property>
                  <property name="widgetResizable">
                   <bool>true</bool>
                  </property>
                  <widget class="QWidget" name="inProgressScrollAreaWidgetContents">
                   <property name="geometry">
                    <rect>
                     <x>0</x>
                     <y>0</y>
                     <width>168</width>
                     <height>419</height>
                    </rect>
                   </property>
                   <layout class="QVBoxLayout" name="inProgressScrollAreaWidgetContentsVerticalLayout"/>
                  </widget>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="readyForTestingGroupBox">
               <property name="font">
                <font>
                 <family>GOST type B</family>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="title">
                <string>Ready for testing</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
               <layout class="QVBoxLayout" name="readyForTestingGroupBoxVerticalLayout">
                <property name="leftMargin">
                 <number>5</number>
                </property>
                <property name="topMargin">
                 <number>5</number>
                </property>
                <property name="rightMargin">
                 <number>5</number>
                </property>
                <property name="bottomMargin">
                 <number>5</number>
                </property>
                <item>
                 <widget class="QScrollArea" name="readyForTestingScrollArea">
                  <property name="frameShape">
                   <enum>QFrame::Panel</enum>
                  </property>
                  <property name="widgetResizable">
                   <bool>true</bool>
                  </property>
                  <widget class="QWidget" name="readyForTestingScrollAreaWidgetContents">
                   <property name="geometry">
                    <rect>
                     <x>0</x>
                     <y>0</y>
                     <width>168</width>
                     <height>419</height>
                    </rect>
                   </property>
                   <layout class="QVBoxLayout" name="readyForTestingScrollAreaWidgetContentsVerticalLayout"/>
                  </widget>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="doneGroupBox">
               <property name="font">
                <font>
                 <family>GOST type B</family>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="title">
                <string>Done</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
               <layout class="QVBoxLayout" name="doneGroupBoxVerticalLayout">
                <property name="leftMargin">
                 <number>5</number>
                </property>
                <property name="topMargin">
                 <number>5</number>
                </property>
                <property name="rightMargin">
                 <number>5</number>
                </property>
                <property name="bottomMargin">
                 <number>5</number>
                </property>
                <item>
                 <widget class="QScrollArea" name="doneScrollArea">
                  <property name="frameShape">
                   <enum>QFrame::Panel</enum>
                  </property>
                  <property name="widgetResizable">
                   <bool>true</bool>
                  </property>
                  <widget class="QWidget" name="doneScrollAreaWidgetContents">
                   <property name="geometry">
                    <rect>
                     <x>0</x>
                     <y>0</y>
                     <width>168</width>
                     <height>419</height>
                    </rect>
                   </property>
                   <layout class="QVBoxLayout" name="doneScrollAreaWidgetContentsVerticalLayout"/>
                  </widget>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
